### PR TITLE
changes needed for windows

### PR DESCRIPTION
**Pull request by Rechi**
still doesn't compile, but nearly done

### CODE REVIEW

**Comment by notspiff** [comment]
trying to cycle appveyor.

**Comment by Rechi** [comment]
this won't build unless vfs.nfs is added in https://github.com/xbmc/repo-binary-addons

**Comment by notspiff** [comment]
yup, noticed the same..

**Comment by Rechi** [comment]
with the last two commits it finally builds
@notspiff just ignore CI and trust me, I compiled it for 32 and 64 bit windows

File: CMakeLists.txt [hunk 1]
```diff
@@ -24,6 +24,10 @@ set(NFS_SOURCES src/NFSConnection.cpp
 
 set(DEPLIBS ${NFS_LIBRARIES} ${p8-platform_LIBRARIES} ${kodiplatform_LIBRARIES})
 
+if(WIN32)
+  list(APPEND DEPLIBS ws2_32)
```
Comment (notspiff) [inline CMakeLists.txt]: this is "all over". is it caused by the above libraries or nfs in this case?

Comment (Rechi) [reply CMakeLists.txt]: You are right, it should be moved to `FindNFS.cmake`.
Just didn't think about as I only wanted to get it compiled.

Comment (notspiff) [reply CMakeLists.txt]: right, so it is nfs and not the platform libraries. hoped we could kill more birds ;)

Comment (Rechi) [reply CMakeLists.txt]: fixed now

**Pull request merged**